core/src/main/kotlin/org/evomaster/core/search/gene/sql/SqlStrings.kt

```diff
@@ -3,4 +3,23 @@ package org.evomaster.core.search.gene.sql
 object SqlStrings {
     const val SINGLE_APOSTROPHE_PLACEHOLDER = "SINGLE_APOSTROPHE_PLACEHOLDER"
 
+    private val QUOTATION_MARK = "\""
+
+    fun removeEnclosedQuotationMarks(str: String): String {
+        return if (str.startsWith(QUOTATION_MARK) && str.endsWith(QUOTATION_MARK)) {
+            str.subSequence(1, str.length - 1).toString()
+        } else {
+            str
+        }
+    }
+
+    private fun encloseWithSingleApostrophePlaceHolder(str: String) = SINGLE_APOSTROPHE_PLACEHOLDER + str + SINGLE_APOSTROPHE_PLACEHOLDER
+
+    fun replaceEnclosedQuotationMarksWithSingleApostrophePlaceHolder(str: String): String {
```

Comment:
maybe move to `GeneUtils`? we are going do some refactoring on escaping at a certain point